Bottom: 2f50de65e0d7153394370c367df423c7371dbcb7
Top:    477303bbeccdf3cd3799b5729428253be585b44a
Author: Jeff Squyres <jsquyres@cisco.com>
Date:   2013-07-16 23:59:47 +0200

autogen.sh: Use autoreconf in autogen.sh

The old sequence of Autotools commands listed in autogen.sh is no
longer correct.  Instead, just use the single "autoreconf" command,
which will invoke all the Right Autotools commands in the correct
order.

Signed-off-by: Jeff Squyres <jsquyres@cisco.com>
Signed-off-by: Yann Droneaud <ydroneaud@opteya.com>
Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/autogen.sh b/autogen.sh
index f433312..24c0a76 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -2,8 +2,4 @@
 
 set -x
 test -d ./config || mkdir ./config
-aclocal -I config
-libtoolize --force --copy
-autoheader
-automake --foreign --add-missing --copy
-autoconf
+autoreconf -ifv -I config
